src/include/baselib/messaging/BrokerErrorCodes.h

```diff
@@ -75,6 +75,16 @@ namespace bl
                 return false;
             }
 
+            static auto tryGetExpectedErrorMessage( SAA_in const bl::eh::error_code& ec ) NOEXCEPT -> const std::string
```

Comment:
Const for a return type without using reference doesn't make sense. The return value should be const std::string& and the return values should be declared as global / const static std::string members of the class.